openshift/README.md

```diff
@@ -0,0 +1,66 @@
+OpenShift 3 Jenkins
+=========================
+This sample walks through the process of starting up an OpenShift cluster and deploying a Jenkins Pod in it.
+
+Steps
+-----
+
+1. Unless you have built OpenShift locally, be sure  to grab the [latest oc command](https://github.com/openshift/origin/releases/latest)
```

Comment:
Btw, https://github.com/openshift/origin/releases/latest goes to oc tools 1.2.2 since time-wise it came after 1.3.1.
But 1.2.2 won't work since `cluster subcommand` is available on 1.3+

Maybe change link to https://github.com/openshift/origin/releases/ and mention "you will need 1.3+"